content/packages/ra-mirrors.txt

```diff
@@ -11,3 +11,4 @@ http://mirrors.ustc.edu.cn/anthon/openra/ra-packages.zip
 http://mirror.oss.maxcdn.com/anthonos/openra/ra-packages.zip
 http://81.169.246.181/ra-packages.zip
 http://zoners.eu/openra/ra-packages.zip
+http://178.63.32.174/ra-music.zip
```

Comment:
These aren't the game assets!

This will be why we have been recieving sporadic reports over the last few months about people having to download the assets multiple times.